From ef5a17911af561c5e8f648fd2d4326945999b51c Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Sat, 22 Jun 2019 11:46:55 +0200
Subject: [PATCH] Update sdm845.c

fixing techpack/audio/asoc/sdm845.c:33:10: fatal error: device_event.h: No such file or directory
---
 techpack/audio/asoc/sdm845.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/techpack/audio/asoc/sdm845.c b/techpack/audio/asoc/sdm845.c
index 4a814747722d..93f05a8447d8 100644
--- a/techpack/audio/asoc/sdm845.c
+++ b/techpack/audio/asoc/sdm845.c
@@ -30,7 +30,7 @@
 #include <sound/jack.h>
 #include <sound/pcm_params.h>
 #include <sound/info.h>
-#include <device_event.h>
+#include "device_event.h"
 #include <dsp/audio_notifier.h>
 #include <dsp/q6afe-v2.h>
 #include <dsp/q6core.h>
